src/app/delegate/page.tsx

```diff
@@ -42,13 +41,12 @@ function RegisterCtaCard() {
       <div className="space-y-2">
         <h3 className="font-serif text-xl sm:text-2xl">Passionate about Celo governance?</h3>
         <p className="text-sm sm:text-base">
-          If you would like to be included in this list, open a pull-request to add your information
-          on Github.
+          If you would like to be included in this list, fill a form by clicking the button.
```

Comment:
nit: wording maybe? I don't find a good replacement but somehow it sounds weird. 🤷 

```suggestion
          If you would like to be included in this list, click the button below and follow the instructions.
```

Reply:
```suggestion
          If you would like to be included in this list,  follow the link below and follow the instructions.

Reply:
yeah it does, solved it IRL with @aaronmgdr